src/Admin/Content/StaticContentAdmin.php

```diff
@@ -11,22 +11,41 @@
 
 namespace Symfony\Cmf\Bundle\SonataPhpcrAdminIntegrationBundle\Admin\Content;
 
+use Ivory\CKEditorBundle\Form\Type\CKEditorType;
 use Sonata\AdminBundle\Datagrid\DatagridMapper;
 use Sonata\AdminBundle\Datagrid\ListMapper;
 use Sonata\AdminBundle\Form\FormMapper;
 use Symfony\Cmf\Bundle\ContentBundle\Model\StaticContentBase;
 use Symfony\Cmf\Bundle\SonataPhpcrAdminIntegrationBundle\Admin\AbstractAdmin;
 use Symfony\Cmf\Bundle\TreeBrowserBundle\Form\Type\TreeSelectType;
-use Symfony\Component\Form\Extension\Core\Type\TextType;
 use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
 
 class StaticContentAdmin extends AbstractAdmin
 {
     protected $translationDomain = 'CmfSonataPhpcrAdminIntegrationBundle';
 
+    /**
+     * Configuration, that can be passed to CKEditorType.
+     *
+     * @var []
+     */
+    private $ckEditorConfig;
+
     public function getExportFormats()
     {
-        return array();
+        return [];
+    }
+
+    /**
+     * The configuration array is explained in the IvoryCKEditorBundle
```

Comment:
i was suggesting this as a second section of the phpdoc - styleci is still not happy with this (i restarted it now that aws works again)

i would keep the first line as "Set configuration for CKEditorType." or something like that